experimenter/experimenter/nimbus-ui/src/styles/index.scss

```diff
@@ -204,6 +204,18 @@ select.form-control.is-warning {
     border-color: $red;
   }
 }
+
+.feature-configs-container {
+  flex-grow: 1;
+}
+
+.feature-configs-count {
+  align-self: center;
+  text-align: center;
+  margin-left: 1rem;
+  min-width: 5rem;
+}
+
```

Comment:
we can use bootstrap classes to achieve this and won't need to define CSS in here